builtin/common/misc_helpers.lua

```diff
@@ -587,6 +587,14 @@ function core.colorize(color, message)
 	return table.concat(lines, "\n") .. core.get_color_escape_sequence("#ffffff")
 end
 
+function core.colorize_chatcommand(cmd, param)
+	if param ~= nil and param ~= "" then
+		return core.colorize(core.COLOR_COMMAND, cmd) .. " " 
+		.. core.colorize(core.COLOR_PARAM, param)
+	else
```

Comment:
Early return = superfluous `else` case
```suggestion
	end
```
And drop the `end` below.